### PR TITLE
chore: update metadata files

### PR DESCRIPTION
**Pull request by frederickfogerty**
As discussed offline, we had duplicated metadata files, and the file inside the `meta` folder was not necessary.

### CODE REVIEW

File: metadata/imgix_data/system-objecttype-extensions.xml [hunk 1]
```diff
@@ -2,28 +2,28 @@
 <metadata xmlns="http://www.demandware.com/xml/impex/metadata/2006-10-31">
     <type-extension type-id="SitePreferences">
         <custom-attribute-definitions>
-            <attribute-definition attribute-id="imgixPageDesignerAPIkey">
+			<attribute-definition attribute-id="imgixapiKey">
```
Comment (luqven) [inline metadata/imgix_data/system-objecttype-extensions.xml:5]: Nit: is there a reason why this key is not camelCased that I'm not aware of?

File: metadata/imgix_data/system-objecttype-extensions.xml [hunk 2]
```diff
@@ -2,28 +2,28 @@
...
             </attribute-definition>
-        </custom-attribute-definitions>
+		</custom-attribute-definitions>
         <group-definitions>
-            <attribute-group group-id="imgixPageDesigner">
-                <display-name xml:lang="x-default">imgix Page Designer</display-name>
-                <attribute attribute-id="imgixPageDesignerSourceName"/>
-                <attribute attribute-id="imgixPageDesignerAPIkey"/>
+            <attribute-group group-id="ImgixAttributes">
+                <display-name xml:lang="x-default">IMGIX Attributes</display-name>
```
Comment (luqven) [inline metadata/imgix_data/system-objecttype-extensions.xml:24]:
```suggestion
                <display-name xml:lang="x-default">imgix Attributes</display-name>
```

File: metadata/imgix_data/system-objecttype-extensions.xml [hunk 3]
```diff
@@ -2,28 +2,28 @@
...
                 <type>string</type>
                 <mandatory-flag>false</mandatory-flag>
                 <externally-managed-flag>false</externally-managed-flag>
                 <min-length>0</min-length>
             </attribute-definition>
-            <attribute-definition attribute-id="imgixPageDesignerSourceName">
-                <display-name xml:lang="x-default">imgix Origin Name</display-name>
-                <description xml:lang="x-default">The Origin Name of your imgix account</description>
+			<attribute-definition attribute-id="imgixDefaultParams">
+                <display-name xml:lang="x-default">Imgix Default Params</display-name>
```
Comment (luqven) [inline metadata/imgix_data/system-objecttype-extensions.xml:14]:
```suggestion
                <display-name xml:lang="x-default">imgix Default Params</display-name>
```

**Review by luqven — COMMENTED** [review]
nit, imgix should never be capitalized

**Comment by frederickfogerty** [comment]
@luqven you make some good points - this PR was just a straight copy but I think it's a good time to make some updates to this

**Comment by frederickfogerty** [comment]
@luqven how is this looking now?

File: metadata/imgix_data/system-objecttype-extensions.xml [hunk 4]
```diff
@@ -2,28 +2,28 @@
 <metadata xmlns="http://www.demandware.com/xml/impex/metadata/2006-10-31">
     <type-extension type-id="SitePreferences">
         <custom-attribute-definitions>
-            <attribute-definition attribute-id="imgixPageDesignerAPIkey">
+			<attribute-definition attribute-id="imgixApiKey">
```
Comment (luqven) [inline metadata/imgix_data/system-objecttype-extensions.xml:5]: Nit, lets stick to spaces and not tabs
```suggestion
                        <attribute-definition attribute-id="imgixApiKey">
```

File: metadata/imgix_data/system-objecttype-extensions.xml [hunk 5]
```diff
@@ -2,28 +2,28 @@
...
+                <description xml:lang="x-default">The API Key of imgix account</description>
                 <type>string</type>
                 <mandatory-flag>false</mandatory-flag>
                 <externally-managed-flag>false</externally-managed-flag>
                 <min-length>0</min-length>
             </attribute-definition>
-            <attribute-definition attribute-id="imgixPageDesignerSourceName">
-                <display-name xml:lang="x-default">imgix Origin Name</display-name>
-                <description xml:lang="x-default">The Origin Name of your imgix account</description>
+			<attribute-definition attribute-id="imgixDefaultParams">
```
Comment (luqven) [inline metadata/imgix_data/system-objecttype-extensions.xml:13]: Some as above
```suggestion
                        <attribute-definition attribute-id="imgixDefaultParams">
```

File: metadata/imgix_data/system-objecttype-extensions.xml [hunk 6]
```diff
@@ -2,28 +2,28 @@
...
+                <display-name xml:lang="x-default">imgix Default Params</display-name>
+                <description xml:lang="x-default">imgix Default Params to be passed in URL</description>
                 <type>string</type>
                 <mandatory-flag>false</mandatory-flag>
                 <externally-managed-flag>false</externally-managed-flag>
                 <min-length>0</min-length>
             </attribute-definition>
-        </custom-attribute-definitions>
+		</custom-attribute-definitions>
```
Comment (luqven) [inline metadata/imgix_data/system-objecttype-extensions.xml:21]: One more
```suggestion
                </custom-attribute-definitions>
```

**Review by luqven — APPROVED** [review]
Lgtm.

If we can change those tabs for spaces great, otherwise not a big deal.

**Comment by frederickfogerty** [comment]
@luqven your wish is my command

**Review by luqven — APPROVED** [review]
lgtm 🚀

**Pull request merged**